tests/src/ViewModel/SocialMediaSharersTest.php

```diff
@@ -19,8 +19,8 @@ public function it_has_data()
             ],
             'encoded' => [
                 'facebookUrl' => 'https://facebook.com/sharer/sharer.php?u=https%3A%2F%2Fexample.com%2Fsome-article-url',
-                'twitterUrl' => 'https://twitter.com/intent/tweet/?text=Some+article+title&url=https%3A%2F%2Fexample.com%2Fsome-article-url',
-                'emailUrl' => 'mailto:?subject=Some+article+title&body=https%3A%2F%2Fexample.com%2Fsome-article-url',
+                'twitterUrl' => 'https://twitter.com/intent/tweet/?text=Some%20article%20title&url=https%3A%2F%2Fexample.com%2Fsome-article-url',
```

Comment:
It seems HTML tags like `<sup>` are stripped elsewhere, so no need to test/spec them here. I assume that 
```
Otoferlin acts as a Ca<sup>2+</sup>
Otoferlin acts as a Ca2+
```
is fine as that what copy/paste does.

To further specify this behavior, we could add these titles extracted for coverage of usual UTF-8 corner cases:
- `Stimulation of hair follicle stem cell proliferation through an IL-1 dependent activation of γδT-cells`
- `3.3-Å resolution cryo-EM structure of human ribonucleotide`
- `Aicardi–Goutières syndrome`
- `the ‘microturbellarian’ roots
- `Michaelis–Menten`
They all seem fine in the current implementation.